JobConfig/pions/PionsStage1_2020.fcl

```diff
@@ -0,0 +1,146 @@
+    GeometryService        : { inputFile      : "JobConfig/common/geom_baseline.txt"    }
+   
+    ConditionsService      : { conditionsfile : "Mu2eG4/test/conditions_01.txt"      }
+    GlobalConstantsService : { inputFile      : "Mu2eG4/test/globalConstants_01.txt" }
+    G4Helper               : {}
+    SeedService            : @local::automaticSeeds
+}
+
+physics : {
+
+    producers : {
+
+        generate: @local::PrimaryProtonGun
+
+        genCounter : { module_type : GenEventCounter }
+
+        g4run : {
+
+            module_type       : Mu2eG4
+            physics           : @local::mu2eg4DefaultPhysics
+            ResourceLimits    : @local::mu2eg4DefaultResourceLimits
+            TrajectoryControl : @local::mu2eg4NoTrajectories
+            debug             : @local::mu2eg4DefaultDebug
+            visualization     : @local::mu2eg4NoVisualization
+
+            generatorModuleLabel : generate
+
+            SDConfig : {
+                enableSD : [ virtualdetector ] // activate just the explicitly listed SDs
+                TimeVD   : { times : [] }
+            }
+
+            // Mu2eG4 cuts are evaluated in the listed order.
+            // The first match wins.  For the "intersection"
+            // type, the first false stops the evaluation.
+            // For the "union" type the first true stops the evaluation.
+            Mu2eG4StackingOnlyCut : @local::mu2eg4CutNeutrinos
+            Mu2eG4SteppingOnlyCut : @local::mu2eg4NoCut
+            Mu2eG4CommonCut : {
+                type : union
+                pars : [
+                    @local::CutEPPhN,
+                    { type: inVolume pars: [ HallAir ] },
+                    {
+                        type  : inVolume
+                        pars  : [ TS3Vacuum, TS4Vacuum, TS5Vacuum, DS1Vacuum, DS2Vacuum, DS3Vacuum ]
+                        write : vacuuas1
+                    }
+                ]
+            }
+        } // g4run
+
+        compressPV : {
+            module_type    : CompressPhysicalVolumes
+            volumesInput   : "g4run"
+            hitInputs      : []
+            particleInputs : [ "g4filter" ]
+        }
+    } // producers
+
+    analyzers : {
+
+        genCountLogger : { module_type : GenEventCountReader }
+
+        collectionSizes : {
+            module_type              : CollectionSizeAnalyzer
+            userModuleLabel          : true
+            useInstanceName          : true
+            useProcessName           : false
+        }
+    } // analyzers
+
+    filters : {
+        g4filter : {
+            module_type               : FilterG4Out
+            mainHitInputs             : [ "g4run:vacuuas1" ]
+            extraHitInputs            : [ "g4run:virtualdetector" ]
+            vetoDaughters             : []
+        }
+    } // filters
+
+    trigFilter    : [ generate, genCounter, g4run, g4filter, compressPV ]
+    trigger_paths : [ trigFilter ]
```

Comment:
Please check with Dave to learn if compressPV is still part of the standard workflows.  If it is not, then you should remove it unless you have a good reason to keep it.